data/human/human missions.txt

```diff
@@ -8295,6 +8295,9 @@ mission "Timothy Radrickson 5a: Timshank Redemption"
 			`	"Now!" She cuts him off, and watches as he sullenly slumps away.`
 			`	Without looking your way she continues, "It was a pleasure to see you again, Captain <last>, but it's time you were leaving. The governor needs you, and my own glorious revolution needs some attention."`
 			`	You leave the dilapidated mines with a bomb, a pistol, and a plan.`
+			action
+				log `I met Union-Colonel Wynnfrith, Timothy Radickson's old secretary on Rand. She appears to have become a revolutionary since I last saw her. I showed her Timothy's list, and she said I could find the survival suit on Dancer, the medication on New Britian, and the Fake ID on Shaula. She provided me with the gun and bomb Timothy asked for.`
```

Comment:
```suggestion
				log `Met Union-Colonel Wynnfrith, Timothy Radickson's old secretary on Rand. She appears to have become a revolutionary since I last saw her. Showed her Timothy's list, and she said I could find a survival suit on Dancer, medication on New Britain, and a fake ID on Shaula. Provided me with the gun and bomb Timothy asked for.`
```

Reply:
Made these changes.